Dockerfile

```diff
@@ -1,15 +1,27 @@
-FROM node
-MAINTAINER jaga santagostino <kandros5591@gmail.com>
-
+FROM node:8 as base
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
+EXPOSE 8000
 
-COPY package.json /usr/src/app
+FROM base as development
+ENV NODE_ENV development
+COPY package.json package-lock.json ./
 RUN npm install
-COPY . /usr/src/app
+COPY .babelrc .eslintignore .eslintrc index.js nodemon.json webpack.config.babel.js webpack.config.dev.js webpack.config.prod.js webpack.config.server.js ./
+COPY client ./client
+COPY config ./config
```

Comment:
Haha so I'm totally nitpicking my own code here… we don't actually need `config/` in the Dockerfile, as it's meant for local machine dev'ing, right?